PyZAPI/production.ini.example

```diff
@@ -0,0 +1,59 @@
+###
```

Comment:
Do we need separate development.ini.example and production.ini.example? Are they significantly different? The answer could be yes, I just want to double check.